### DISCUSSION TITLE
ModuleNotFoundError: No module named 'omni.kit.usd'

### DISCUSSION THREAD

**Discussion by mengxyokok** [StanfordVL/BEHAVIOR-1K#955] [Q&A] [answered]
When I use Isaac SIm 2023.1.1, it is ok. but when I use Isaac SIm 4.2.0 , the problem comes
how could I  solve this problem?

**Comment by hang-yin** [comment, marked as answer]
Hi! Thanks for reaching out. OmniGibson v1.1.0 was released yesterday, please check [here](https://github.com/StanfordVL/OmniGibson/releases/tag/v1.1.0) for our release notes and [here](https://behavior.stanford.edu/omnigibson/getting_started/installation.html) for our updated documentation on how to install (& more!) - note that this release is compatible with Isaac Sim 4.0.0 and 4.1.0 but not 4.2.0 yet. Let us know here if you run into any issues. Thanks!

**Reply by muye1202** [reply]
Hi Yin

I'm trying to use OmniGibson 4.1.0.0 and run the ReKep repo from your lab, it still gives me error saying cannot find omni.kit.usd, wondering what missing packages I should install?

thanks!
